packages/http-client-csharp/package.json

```diff
@@ -46,29 +46,29 @@
     "json-serialize-refs": "0.1.0-0"
   },
   "peerDependencies": {
-    "@azure-tools/typespec-azure-core": ">=0.49.0 <1.0.0 || ~0.50.0-0",
-    "@azure-tools/typespec-client-generator-core": ">=0.49.0 <1.0.0 || ~0.50.0-0",
-    "@typespec/compiler": ">=0.63.0 <1.0.0 || ~0.64.0-0",
-    "@typespec/http": ">=0.63.0 <1.0.0 || ~0.64.0-0",
-    "@typespec/openapi": ">=0.63.0 <1.0.0 || ~0.64.0-0",
-    "@typespec/rest": ">=0.63.0 <1.0.0 || ~0.64.0-0",
-    "@typespec/versioning": ">=0.63.0 <1.0.0 || ~0.64.0-0"
+    "@azure-tools/typespec-azure-core": ">=0.50.0 <1.0.0 || ~0.51.0-0",
+    "@azure-tools/typespec-client-generator-core": ">=0.50.0 <1.0.0 || ~0.51.0-0",
+    "@typespec/compiler": ">=0.64.0 <1.0.0 || ~0.65.0-0",
+    "@typespec/http": ">=0.64.0 <1.0.0 || ~0.65.0-0",
+    "@typespec/openapi": ">=0.64.0 <1.0.0 || ~0.65.0-0",
+    "@typespec/rest": ">=0.64.0 <1.0.0 || ~0.65.0-0",
+    "@typespec/versioning": ">=0.64.0 <1.0.0 || ~0.65.0-0"
```

Comment:
shall we consider remove the second part `|| ~0.65.0-0`?
according to @allenjzhang , it is not actually working.

Reply:
This is used in nightly build, whose version is 0.65.XXX.

Reply:
This is useful for nightly build tsp-next dev versions.